examples/time_frequency/plot_compute_raw_data_spectrum.py

```diff
@@ -70,3 +71,16 @@
 
 ax.set_title('Four left-temporal magnetometers')
 plt.legend(['Without SSP', 'With SSP', 'SSP + Notch'])
+
+# Alternatively, you may also create PSDs from Raw objects with psd_XXX
+kws_psd = dict(tmin=tmin, tmax=tmax, fmin=fmin, fmax=fmax, n_jobs=1)
+psds_wel, freqs_wel = psd_welch(raw, n_fft=n_fft, **kws_psd)
+psds_mt, freqs_mt = psd_multitaper(raw, low_bias=True, **kws_psd)
```

Comment:
no these are all really good points and they've been sticking points when I've taught python stuff to people before, I'll keep a new user in mind when working on examples from now on